### PR TITLE
SecretManager bump version v1beta1->v1

### PR DESCRIPTION
**Pull request by gwhitehawk**
Client library bumped: https://packagist.org/packages/google/cloud-secret-manager

### CODE REVIEW

**Comment by gwhitehawk** [comment]
Can someone please look at the PR?

**Comment by sethvargo** [comment]
Hey @dwsupplee - do you have time to look at this?

**Comment by dwsupplee** [comment]
@sethvargo, @gwhitehawk - yes I can review. I've put time on my calendar for Monday next week.

**Comment by dwsupplee** [comment]
@gwhitehawk / @sethvargo - it looks like the failures in the tests are due to the fact V1beta1 -> V1 changed the `Secret` instance from being optional to required. Here's the diff in the signature:

```diff
- public function createSecret($parent, $secretId, array $optionalArgs = [])
+ public function createSecret($parent, $secretId, $secret, array $optionalArgs = [])
```

Once that is updated, everything else LGTM and I suspect the tests will pass.

**Comment by gwhitehawk** [comment]
Hmm, I don't think any place in this sample where the client function signature would be violated (quickstart.php, src/create_secret.php, test/secretmanagerTest.php). Also it's hard to see what's wrong from the log (from github checks).

**Comment by dwsupplee** [comment]
Unless there is another change in a signature, I think things should be good to go if you use the following in these 3 locations:

```php
$secret = $client->createSecret(
    $parent,
    $secretId, 
    new Secret([ 
        'replication' => new Replication([ 
            'automatic' => new Automatic()
        ])
    ])
 ); 
```

https://github.com/GoogleCloudPlatform/php-docs-samples/blob/aec692f7d24eef7f13c9cb1c4b2209ef2a0db6b0/secretmanager/quickstart.php#L45-L52

https://github.com/GoogleCloudPlatform/php-docs-samples/blob/aec692f7d24eef7f13c9cb1c4b2209ef2a0db6b0/secretmanager/src/create_secret.php#L50-L57

https://github.com/GoogleCloudPlatform/php-docs-samples/blob/aec692f7d24eef7f13c9cb1c4b2209ef2a0db6b0/secretmanager/test/secretmanagerTest.php#L82-L88

**Comment by sethvargo** [comment]
@gwhitehawk I opened https://github.com/gwhitehawk/php-docs-samples/pull/1 against your fork to make those changes. If you merge that, they should appear here and CI will pass.

**Comment by gwhitehawk** [comment]
Oh sorry @dwsupplee, I thought you meant that instead of specifying secret in the calls, the default one is used. I fixed it now according to the suggestion.

File: secretmanager/quickstart.php [hunk 1]
```diff
@@ -43,13 +43,13 @@
...
 
 // Create the parent secret.
-$secret = $client->createSecret($parent, $secretId, [
-    'secret' => new Secret([
+$secret = $client->createSecret($parent, $secretId,
+  new Secret([
         'replication' => new Replication([
             'automatic' => new Automatic(),
         ]),
     ]),
```
Comment (dwsupplee) [inline secretmanager/quickstart.php:51]:
```suggestion
    ])
```

File: secretmanager/src/create_secret.php [hunk 2]
```diff
@@ -48,13 +48,13 @@
...
 
 // Create the secret.
-$secret = $client->createSecret($parent, $secretId, [
-    'secret' => new Secret([
+$secret = $client->createSecret($parent, $secretId,
+    new Secret([
         'replication' => new Replication([
             'automatic' => new Automatic(),
         ]),
     ]),
```
Comment (dwsupplee) [inline secretmanager/src/create_secret.php:56]:
```suggestion
    ])
```

File: secretmanager/test/secretmanagerTest.php [hunk 3]
```diff
@@ -79,13 +79,13 @@ private static function createSecret(): Secret
...
         $secretId = self::randomSecretId();
 
-        return self::$client->createSecret($parent, $secretId, [
-            'secret' => new Secret([
+        return self::$client->createSecret($parent, $secretId,
+            new Secret([
                 'replication' => new Replication([
                     'automatic' => new Automatic(),
                 ]),
             ]),
```
Comment (dwsupplee) [inline secretmanager/test/secretmanagerTest.php:87]:
```suggestion
            ])
```

**Comment by gwhitehawk** [comment]
Is this failure relevant? I can't tell what went wrong: https://source.cloud.google.com/results/invocations/8013d459-c058-45ae-928a-379e6f652fb1/targets/cloud-devrel%2Fphp-docs-samples%2Fpresubmit/log

**Comment by dwsupplee** [comment]
@gwhitehawk the failure looks like it may be related to the following PR: https://github.com/GoogleCloudPlatform/php-docs-samples/pull/1052/files

@bshaffer is this one okay to merge?

**Comment by sethvargo** [comment]
If it helps, I just ran the SM tests locally:

```
$ ../testing/vendor/bin/phpunit  --verbose --debug
PHPUnit 7.5.20 by Sebastian Bergmann and contributors.

Runtime:       PHP 7.4.4
Configuration: /Users/sethvargo/Development/samples/php-docs-samples/secretmanager/phpunit.xml.dist
Error:         No code coverage driver is available

Test 'quickstartTest::testQuickstart' started
Test 'quickstartTest::testQuickstart' ended
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testAccessSecretVersion' started
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testAccessSecretVersion' ended
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testAddSecretVersion' started
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testAddSecretVersion' ended
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testCreateSecret' started
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testCreateSecret' ended
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testDeleteSecret' started
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testDeleteSecret' ended
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testDestroySecretVersion' started
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testDestroySecretVersion' ended
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testDisableSecretVersion' started
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testDisableSecretVersion' ended
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testEnableSecretVersion' started
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testEnableSecretVersion' ended
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testGetSecretVersion' started
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testGetSecretVersion' ended
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testGetSecret' started
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testGetSecret' ended
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testIamGrantAccess' started
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testIamGrantAccess' ended
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testIamRevokeAccess' started
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testIamRevokeAccess' ended
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testListSecretVersions' started
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testListSecretVersions' ended
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testListSecrets' started
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testListSecrets' ended
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testUpdateSecret' started
Test 'Google\Cloud\Samples\SecretManager\secretmanagerTest::testUpdateSecret' ended


Time: 12.21 seconds, Memory: 8.00 MB

OK (15 tests, 18 assertions)
```

**Comment by dwsupplee** [comment]
Yeah the system tests are fine, it is just a cs check which runs only on 7.3. I think this should be okay to merge, just want to confirm with @bshaffer.

**Comment by bshaffer** [comment]
Yep! That CS issue was fixed in #1052

**Comment by dwsupplee** [comment]
Thanks

**Pull request merged**